tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -57,6 +57,7 @@ init_env() {
 			LIBC="gnu"
 			ARCH="powerpc64le"
 			ARCH_LIBC=${ARCH}-linux-${LIBC}
+			extra_rust_flags=""
```

Comment:
Hi @Amulyam24, is this line necessary?

Reply:
Hi @cmaf, yes it is necessary because the extra rust flag `-C link-self-contained=yes` is no more supported on the ppc64le rust target which is causing the failure while compiling virtiofsd
```
error: option `-C link-self-contained` is not supported on this target
```